default.py

```diff
@@ -24,38 +27,22 @@
 addon_settings = xbmcaddon.Addon(id='script.lutris')
 # Get the addon name from addon settings
 addon_name = addon_settings.getAddonInfo('name')
+# Plugin User Data Folder
+userdata = addon_settings.getAddonInfo('path')
```

Comment:
Should be decoded from UTF-8. See [Add-on unicode paths](https://kodi.wiki/view/Add-on_unicode_paths) for more information on how Kodi handles unicode.

Reply:
Thanks for the resource.. Didn't know about that but makes sense if path contains other chars